src/components/UI/Spinner/Spinner.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import EnzymeAdapter from 'enzyme-adapter-react-16';
+
+import Spinner from './Spinner';
+
+Enzyme.configure({ adapter: new EnzymeAdapter() });
+
+describe('Testing of <Spinner />  componenet', () => {
+    test("render an spinner", () => {
+        const wrapper = shallow(<Spinner />);
+       expect(wrapper).toBeTruthy();
```

Comment:
Wrapper will always be truthy. Write more test cases specific to element, class, states etc